foodnet/membership/models.py

```diff
@@ -12,50 +19,73 @@ class UserProfile(models.Model):
         (FEMALE, 'female'),
         (MALE, 'male')
     )
-    
-    user = models.OneToOneField(User)
-    middlename = models.CharField(max_length=255)
-    
+
+    user = models.OneToOneField(User, editable=False)
+    middle_name = models.CharField(max_length=30, null=True)
+
     # old system: adr1, adr2, streetno, floor, door
-    address = models.TextField(max_length=2000)
-    postcode = models.CharField(max_length=255)
-    city = models.CharField(max_length=255)
-    tel = models.CharField(max_length=255)
-    tel2 = models.CharField(max_length=255)
+    address = models.TextField(max_length=255)
+    postcode = models.CharField(max_length=30)
+    city = models.CharField(max_length=50)
+    tel = models.CharField(max_length=15)
+    tel2 = models.CharField(max_length=15, null=True)
     sex = models.CharField(max_length=1, choices=SEX_CHOICES)
-    birthday = models.DateField(null=True)
+    dob = models.DateField(null=True)  # old system: birthday
```

Comment:
Again, I'd prefer the more verbose `date_of_birth`. Also, is this a requirement for the system? Have we thought about the users privacy in this matter? I believe we can go a long way without knowing peoples `date_of_birth` :)